ui/packages/ui/src/Pages/Viewer/Components/Warnings.tsx

```diff
@@ -155,17 +148,15 @@ const StaminaWarning = ({ data }: WarningProps) => {
 
   return (
     <DismissibleCallout
-        title="Insufficient Stamina"
+        title="Delay in Dash - Insufficient Stamina"
         intent={Intent.WARNING}
         show={visible}
         onDismiss={() => setShow(false)}>
       <p>
-        An abnormal amount of iterations failed to execute an action because the character did not
-        have enough stamina. Consider updating the config to better manage stamina, as no actions
-        are performed during failures.
+        Some iterations delayed executing dash due to insufficient stamina. This causes the active character to idle until enough stamina regnerated. Consider updating the config if the downtime below is undesired.
```

Comment:
This section is about charged attacks, or any action that consumes stamina. Also fixing typos:

> Some iterations delayed executing a stamina-draining action due to insufficient stamina. This causes the active character to idle until enough stamina is regenerated. Consider updating the config if the downtime below is undesired.